### PR TITLE
Add twitter preview

### PR DESCRIPTION
**Pull request by jinhojang6**
Add a twitter preview which will be shown as below.
Screen Shot 2021-02-10 at 2 35 13 PM

The `content="<%= config.url %>/images/keycard-preview.jpg"` ejs variable will work when this PR is merged with the `master` branch. Twitter needs an absolute url for the preview but cannot access the client side variable for the new image (e.g., window.location.href) in hexo.

### CODE REVIEW

**Review by guylouis — APPROVED** [review]

File: themes/keycard/layout/partial/head.ejs [hunk 1]
```diff
@@ -17,6 +17,21 @@
   <link rel="apple-touch-icon" sizes="152x152" href="<%= url_for('images/apple-touch-icon-152x152-precomposed.png') %>">
   <link rel="apple-touch-icon" sizes="180x180" href="<%= url_for('images/apple-touch-icon-precomposed.png') %>">
   <link rel="apple-touch-icon" href="<%= url_for('images/apple-touch-icon-precomposed.png') %>">
+
+  <meta property="og:site_name" content="Keycard" />
+  <meta property="og:type" content="website" />
+  <meta property="og:title" content="Keycard - Secure, Contactless, Open Source, Crypto Hardwallet" />
+  <meta property="og:description" content="A new type of smartcard built with an open API for simple integration with crypto wallets and hardware. Designed to safely send, store, and receive cryptocurrencies in a seamless contactless experience." />
+  <meta property="og:url" content="https://keycard.tech/" />
```
Comment (jakubgs) [inline themes/keycard/layout/partial/head.ejs:25]: Shouldn't this be `<%= config.url %>` as well?

Comment (jinhojang6) [reply themes/keycard/layout/partial/head.ejs:25]: @jakubgs You mean `<meta property="og:url" content="https://keycard.tech/" />` this? In that case, you are right. type, title, description are okay.

Comment (jakubgs) [reply themes/keycard/layout/partial/head.ejs:25]: Yes, I mean the URL part. It should be dynamic since that's what `config.url` is for.

Comment (jinhojang6) [reply themes/keycard/layout/partial/head.ejs]: @jakubgs Updated to <%= config.url %>, please take a look.

**Review by jakubgs — APPROVED** [review]

**Pull request merged**